components/test_raftstore/src/server.rs

```diff
@@ -429,6 +430,8 @@ impl ServerCluster {
             TokioBuilder::new_multi_thread()
                 .thread_name(thd_name!("debugger"))
                 .worker_threads(1)
+                .after_start_wrapper(|| {})
+                .before_stop_wrapper(|| {})
```

Comment:
For this specific case, I'm wondering if we can simplify the code here by merge these two line into one, such as
.on_start_stop_wrapper().